kratos.gid/apps/DEM/xml/Processes.xml

```diff
@@ -199,12 +199,28 @@
+           <parameter n="AngularPeriod"    parent="true"           pn="Period (s)"            v="5.0"    help="in seconds"/> </parameter>
           <parameter n="AngularStartTime" parent="LinearPeriodic" pn="Start time (s)"        v="0.0"    help="in seconds"/>
           <parameter n="AngularEndTime"   parent="LinearPeriodic" pn="End time (s)"          v="1000.0" help="in seconds"/>
 
+          <parameter n="Ax"             parent="FixedDOFs" type="combo"      v="No"    pn="Fix Velocity X"         values="No,Constant" pvalues="No,Constant" help="Fix Velocity X">
+           <parameter n="Vx"             parent="Constant"                    v="1.0"   pn="   Value (m/s)"                   help="X Velocity"/></parameter>
+          <parameter n="Ay"             parent="FixedDOFs" type="combo"      v="No"    pn="Fix Velocity Y"         values="No,Constant" pvalues="No,Constant" help="Fix Velocity Y">
+           <parameter n="Vy"             parent="Constant"                    v="0.0"   pn="   Value (m/s)"                   help="Y Velocity"/></parameter>
+          <parameter n="Az"             parent="FixedDOFs" type="combo"      v="No"    pn="Fix Velocity Z"         values="No,Constant" pvalues="No,Constant" help="Fix Velocity Z">
+           <parameter n="Vz"             parent="Constant"                    v="0.0"   pn="   Value (m/s)"                   help="Z Velocity"/></parameter>
+          <parameter n="Bx"             parent="FixedDOFs" type="combo"      v="No"    pn="Fix Angular Velocity X" values="No,Constant" pvalues="No,Constant" help="Fix Angular Velocity X">
+           <parameter n="AVx"            parent="Constant"                    v="0.0"   pn="   Value (rad/s)"                 help="X Angular Velocity"/></parameter>
+          <parameter n="By"             parent="FixedDOFs" type="combo"      v="No"    pn="Fix Angular Velocity Y" values="No,Constant" pvalues="No,Constant" help="Fix Angular Velocity Y">
+           <parameter n="AVy"            parent="Constant"                    v="0.0"   pn="   Value (rad/s)"                 help="Y Angular Velocity"/></parameter>
+          <parameter n="Bz"             parent="FixedDOFs" type="combo"      v="No"    pn="Fix Angular Velocity Z" values="No,Constant" pvalues="No,Constant" help="Fix Angular Velocity Z">
+           <parameter n="AVz"            parent="Constant"                    v="0.0"   pn="   Value (rad/s)"                 help="Z Angular Velocity"/></parameter>
+          <parameter n="VStart"         parent="FixedDOFs"                   v="0.0"   pn="Start time (s)"                   help="Start Time"/>
+          <parameter n="VEnd"           parent="FixedDOFs"                   v="100.0" pn="End time (s)"                     help="Start Time"/>
+
           <parameter n="VelocitiesFilename" pn="Filename" parent="FromATable" type="combo" v="velocities.dat"/>
```

Comment:
There is a way to input a file o a tablefile 

file is for regular files
tablefile are csv that can be automaticaly written in the mdpa as table

try setting type="file" instead of combo